packages/hash/api/src/db/adapter.ts

```diff
@@ -463,6 +479,20 @@ export interface DBClient {
     path?: string;
   }): Promise<DBLink[]>;
 
+  /**
+   * Gets all the incoming links of an entity.
+   *
+   * @todo: support getting the incoming links of an entity at
+   * a particular point in its history
```

Comment:
Have created [a task for this](https://app.asana.com/0/1200211978612931/1201959601641869/f)